### ISSUE TITLE
Make it clearer what expectations are being tested in results

### ISSUE THREAD

**Issue by mfairchild365** [accessibilitysupported/a11ysupport.io#116]
As an example, https://a11ysupport.io/tests/apg__checkbox-two-state has two failing results for `aria-labelledby`. Is that expectation for the checkboxes or for the group role?
